src/Sources.jl

```diff
@@ -44,6 +42,6 @@ function getStarkCurves(source::StarkSamplingSource{PT}, n::I) where {PT, I<:Int
     [starkCurves[sample] for sample ∈ stateSamples]
 end
 
-function getSamples(dists::Dict{Symbol, Samp}, n::I) where {Symbol, Samp, I<:Integer}
+function getSamples(dists::Union{Dict{Symbol, Samp}, NamedTuple}, n::I) where {Symbol, Samp, I<:Integer}
```

Comment:
just curious why you made this change?

Reply:
Somehow my comment disappeared, so don't be surprised if it's here twice...
Because Simon used `NamedTuple`s, whilst I used `Dict`s. On the long run, one of both sides should be modified to be unison, but for now the `Union` (which allows both types) is the best I could come up with.
Thanks for reviewing!